cuenca/resources/base.py

```diff
@@ -50,17 +59,19 @@ def refresh(self):
 class Creatable(Resource):
     @classmethod
     def _create(cls, **data) -> Resource:
-        resp = session.post(cls._endpoint, data)
+        resp = session.post(cls._resource, data)
         return cls._from_dict(resp)
 
 
 class Queryable(Resource):
     _query_params: ClassVar = QueryParams
 
+    created_at: dt.datetime
```

Comment:
Although `ApiKey` and `Transfers`  extends of `Queryable`, this field does not exist in these classes.
Only takes fields of `Resource`

Reply:
This should solve that: https://github.com/cuenca-mx/cuenca-python/commit/cdc03955114a9008df14883695fa16d0c4be3127